board_heu.py

```diff
@@ -0,0 +1,70 @@
+import draughts
+import random
+
+from draughts import Board, Move
+
+from game import Game, Player, BOARD_SIZE
+
+"""
+This class is used to get heuristics from a Board (instead of Game).
+To be deleted
```

Comment:
Let's get rid of this. All of this is pretty much the same as in heuristics.py, right?